react-function-components/src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+function customButton() {
+  return <button>Click Me</button>;
+}
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+const button = customButton();
```

Comment:
When using a component we never directly call the component function, we instead write it as JSX.
```JSX
const button = <CustomButton />;
```